fastai/callback/tensorboard.py

```diff
@@ -13,8 +13,9 @@
 # Cell
 class TensorBoardCallback(Callback):
     "Saves model topology, losses & metrics"
-    def __init__(self, log_dir=None, trace_model=True, log_preds=True, n_preds=9):
-        store_attr('log_dir,trace_model,log_preds,n_preds')
+    def __init__(self, log_dir=None, trace_model=True, log_preds=True, n_preds=9, projector=False, layer=None):
+        store_attr('log_dir,trace_model,log_preds,n_preds,projector,layer')
```

Comment:
Since you're storing everything, you can remove the argument entirely and just say `store_attr()`